### PR TITLE
I needed a better diaper :(

### PR DESCRIPTION
**Pull request by seandst**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 205277fe536c0149a06f78b44fc6e22260fd1840:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
I personally believe that the diaper might deserve its own pypi module.

**Comment by seandst** [comment]
haha, true

I'd be surprised if this didn't already exist...

<checking>

**Comment by seandst** [comment]
https://pypi.python.org/pypi?%3Aaction=search&term=diaper&submit=search ?!?!?!

**Comment by seandst** [comment]
I'm making this.

**Comment by dajoRH** [comment]
Lint report for commit 98e0b95fa4b5b5f4108b9c74d406cdae567352e0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Requirements have changed. @seandst , @psav ?
*CFME QE Bot*

File: cfme/fixtures/pytest_selenium.py [hunk 1]
```diff
@@ -242,9 +243,9 @@ def in_flight():
         The keys are: ``jquery, prototype, miq, spinner and document``.
         The values are: ``True`` if running, ``False`` otherwise.
     """
-    try:
+    with diaper:
         return execute_script(js.in_flight)
-    except:  # This diaper is OK (mfalesni)
+    if diaper.smelly:
```
Comment (mfalesni) [inline cfme/fixtures/pytest_selenium.py]: This is epic.

**Comment by dajoRH** [comment]
Lint report for commit 9b432c42d82f6d01d606f41b937f7aa77a7e7682:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/fixtures/pytest_selenium.py (hunk 1 above)
Comment (seandst) [inline cfme/fixtures/pytest_selenium.py]: Thanks to @mhrivnak for  the idea.

Comment (seandst) [inline cfme/fixtures/pytest_selenium.py]: In fact, I'm going to make sure he's one of the authors, too. I'm taking the least amount of credit possible. :D

**Pull request merged**